devops/metabulo/R/analyses.R

```diff
@@ -0,0 +1,35 @@
+#' Basic PCA plot
+#'
+#' Uses MetaboAnalystR's PlotPCAPairSummary method
+
+anova_turkey_adjustment <- function(measurements, groups) {

```

Comment:
I believe this should be "tukey".

Reply:
that happens when you think of turkey to eat :-D